### PR TITLE
patched to build on NetBSD

### PR DESCRIPTION
**Pull request by voidpin**
See issue #2092

### CODE REVIEW

File: doc/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,7 @@
+include(GNUInstallDirs)
+
 file(GLOB man_1 *.1)
-install(FILES ${man_1} DESTINATION share/man/man1)
+install(FILES ${man_1} DESTINATION ${CMAKE_INSTALL_MANDIR]/man/man1)
```
Comment (ghutchis) [inline doc/CMakeLists.txt]: Need { } or this fails.

Comment (voidpin) [reply doc/CMakeLists.txt]: fixed

**Comment by voidpin** [comment]
Sorry about that :(
I'm rather new to all this...
Is it correct now? I've corrected it on my repo.

**Comment by baoilleach** [comment]
Just a note to say that I'm putting together the release notes for 3.0.1. I usually include the github name and people's real names. I don't have yours - contact me asap (my username at gmail.com) if you wish me to include it.

**Pull request merged**